tests/test_portfolio_optimization.py

```diff
@@ -90,13 +92,15 @@ def test_portfolio_qokitandqiskit(simname):
             assert np.allclose(qaoa_obj_qiskit(x0), qaoa_obj_qokit(x0))
 
             qc = get_qaoa_circuit(po_problem, gammas=x0[:p] / 2, betas=x0[p:] / 2, depth=p)
-            result = execute(qc, backend).result()
-            sv1 = reverse_array_index_bit_order(result.get_statevector())
+            # result = transpile(qc, backend).result()
```

Comment:
Can you please remove commented out code?

Reply:
Done!